### PR TITLE
planner: support set binding status by sql digest

### PR DESCRIPTION
**Pull request by fzzf678**
### What problem does this PR solve?

Issue Number: res #39199 

Problem Summary:

### What is changed and how it works?
Support set binding status by sql digest
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: planner/core/planbuilder.go [hunk 1]
```diff
@@ -1003,10 +1003,18 @@ func (b *PlanBuilder) buildDropBindPlan(v *ast.DropBindingStmt) (Plan, error) {
...
+		p = &SQLBindPlan{
+			SQLBindOp:    OpSetBindingStatus,
+			NormdOrigSQL: parser.Normalize(utilparser.RestoreWithDefaultDB(v.OriginNode, b.ctx.GetSessionVars().CurrentDB, v.OriginNode.Text())),
+			Db:           utilparser.GetDefaultDB(v.OriginNode, b.ctx.GetSessionVars().CurrentDB),
+		}
+	} else {
+		p = &SQLBindPlan{
+			SQLBindOp: OpSetBindingStatusByDigest,
+			SQLDigest: v.SQLDigest,
+		}
```
Comment (qw4990) [inline planner/core/planbuilder.go:1017]: How about:
```
if v.OriginNode != nil {...}
else if v.SQLDigist != "" {...}
else {return error}
```

Comment (fzzf678) [reply planner/core/planbuilder.go]: fixed, and PTAL again ^^

**Review by Reminiscent — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]

**Pull request merged**